app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_name, source_file, destination_file, *_ = command.split()
```

Comment:
first you should check that the length of `command.split()` is not less than three